src/main/java/millom/sandbox/mapper/NasaMapper.java

```diff
@@ -0,0 +1,38 @@
+package millom.sandbox.mapper;
+
+import com.fasterxml.jackson.core.JacksonException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import millom.sandbox.pojos.Weather;
+
+public class NasaMapper {
+  private static final ObjectMapper MAPPER = new ObjectMapper();
+  static {
+    MAPPER.enable(DeserializationFeature.ACCEPT_SINGLE_VALUE_AS_ARRAY);
+  }
+
+  public Weather deserializeWeather(String response){
+    try {
+       return MAPPER.readValue(response,Weather.class);
+      //return w;
```

Comment:
Let's get rid of the commented out line.

Reply:
removed commented line from NasaMapper.java

` //return w;`